domain_scoring/domain_scoring.py

```diff
@@ -60,34 +57,15 @@ def _preprocess(self, data: List[Tuple[MetaPath, MetaPath]]) -> List[List[int]]:
 
         return vectors
 
-    def _all_pairs(self, list: List, inverse: bool = False) -> List[Tuple]:
-        """
-
-        :param list: the list from which the elements are taken.
-        :param inverse: if True also the inverse pairs are added.
-        :return: All pairs found in the input list.
-        """
-
-        pairs = []
-        for key in range(len(list)):
-            element = list[key]
-            lower = 0 if inverse else key
-            for successor in list[lower:]:
-                if element is successor:
-                    continue
-
-                pairs.append((element, successor))
-        return pairs
-
-
-    def _transform_to_domain_values(self, inferred_ratings) -> List:
+    def _transform_to_domain_values(self, metapaths_pairs: List[Tuple[MetaPath, MetaPath]], classification: List[int]) -> List:
```

Comment:
Please add `List[float]` for the return value if plausible

Reply:
Added the respective datatype.